doc/TRIAGING.md

```diff
@@ -0,0 +1,146 @@
+# Triage
+
+## Preface
+
+This document elaborates on the triaging process that we use to prioritize and resolve incoming requests, bug reports etc.
```

Comment:
I suppose that at the moment this is pretty much exclusively about what we do with GitHub issues.  Presumably if someone reports an issue through some other channel they should be encouraged to report it on GitHub instead.